### PR TITLE
Feature: Simple Identity Implementation

### PR DESCRIPTION
**Pull request by talekhinezh**

### CODE REVIEW

File: radix-engine-interface/src/address/hrpset.rs [hunk 1]
```diff
@@ -13,8 +13,9 @@ pub struct HrpSet {
 
     normal_component: String,
     account_component: String,
+    identity_component: String,
 
-    system: String,
+    system_component: String,
```
Comment (iamyulong) [inline radix-engine-interface/src/address/hrpset.rs:18]: Should we just remove this?

File: radix-engine/src/engine/kernel.rs [hunk 2]
```diff
@@ -86,7 +86,7 @@ where
                     auth_zone_params.initial_proofs.into_iter().collect(),
                 );
                 let node_id = api.allocate_node_id(RENodeType::AuthZoneStack)?;
-                api.create_node(node_id, RENode::AuthZoneStack(auth_zone))?;
+                api.create_node(node_id, RENodeInit::AuthZoneStack(auth_zone))?;
```
Comment (iamyulong) [inline radix-engine/src/engine/kernel.rs:89]: Liked this rename very much!

**Review by iamyulong — APPROVED** [review]
LGT

File: radix-engine-interface/src/address/hrpset.rs (hunk 1 above)
Comment (talekhinezh) [reply radix-engine-interface/src/address/hrpset.rs:18]: Yup good point!

Comment (talekhinezh) [reply radix-engine-interface/src/address/hrpset.rs:18]: Actually, we are still using this for `EpochManager` and `Clock` and `Validator`. I will delay updating this until we get agreement with product side what's the right hrp to use for these.

**Pull request merged**